app/views/validators/_software_versions.html.erb

```diff
@@ -18,14 +18,14 @@
               <% sort_software_versions(software_versions).each do |group| %>
                 <% if group.first[1].is_a?(Hash) %>
                   <tr>
-                    <td class="<%= group.first[0] == current_software_version(batch, params[:network]) ? 'h4 text-success' : '' %>">
+                    <td class="<%= group.first[0].to_s == current_software_version(batch, params[:network]).to_s ? 'h4 text-success' : '' %>">
```

Comment:
There's no batch variable anymore (see changes you made to controller), so this version is not displayed correctly. You should use batch number saved in report. Check also other references to batch in cluster stats views.

Reply:
here's the batch assingment:
https://github.com/brianlong/www.validators.app/pull/267/files#diff-a777acc0360af07189fc742f4cb67caba42a8db3f17fda4bbdbe79fa37a4c510R9